stock_indicators/_cslib/__init__.py

```
@@ -11,6 +12,7 @@
 import logging
 import platform
 from pathlib import Path
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

(F401)